sdk/spring/spring-cloud-azure-autoconfigure/pom.xml

```diff
@@ -177,7 +177,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-messaging-eventgrid</artifactId>
-      <version>4.18.0</version> <!-- {x-version-update;com.azure:azure-messaging-eventgrid;dependency} -->
+      <version>4.19.0-beta.1</version> <!-- {x-version-update;unreleased_com.azure:azure-messaging-eventgrid;dependency} -->
```

Comment:
Hi @billwert @srnagar, will these changes be changed back after release?
FYI: @saragluna

Reply:
We should not depend on an unreleased version, any update for this? @joshfree @samvaity